test/regression/SqliteSnapshotTest.cpp

```diff
@@ -0,0 +1,62 @@
+#include "../TestConfig.h"
```

Comment:
Did you happen to know what I'm missing to be able to do `#include "test/TestConfig.h" here? If your not sure I'll come to this later on.